pkg/broadcaster/broadcaster.go

```diff
@@ -82,11 +77,11 @@ func (b *Broadcaster) Write(p []byte) (int, error) {
 
 func (b *Broadcaster) Close() error {
 	b.mu.Lock()
-	errs := make([]error, 0, len(b.clients))
-	for c := range b.clients {
-		errs = append(errs, c.Close())
+	for w, ch := range b.clients {
+		close(ch)
+		_ = w.Close() // Best effort.
 	}
 	b.clients = nil
```

Comment:
Make sure all broadcast goroutines stopped before returning from Close as well?

It would be nice if we could do the same in Unsubscribe, but only for that particular client, but I'll take a Close guarantee only.

Reply:
Done.